src/openrct2/peep/GuestPathfinding.cpp

```diff
@@ -142,13 +142,13 @@ static int32_t guest_surface_path_finding(Peep* peep)
     int16_t z = peep->next_z;
     Direction randDirection = scenario_rand() & 3;
 
-    if (!fence_in_the_way(x, y, z, z + 4, randDirection))
+    if (!fence_in_the_way({ x, y, z * 8, (z * 8) + 4 * 8 }, randDirection))
```

Comment:
Can you create a constexpr called `PATH_HEIGHT` and define it as 4 * COORDS_Z_STEP, then use that here?

```suggestion
    if (!fence_in_the_way({ x, y, z * COORDS_Z_STEP, (z * COORDS_Z_STEP) + PATH_HEIGHT }, randDirection))
```